docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
-Enable Nvidia GPUs
+.. _nvidia-gpu:
```

Comment:
what is this line for? Coming from the starter-pack? rst (I am used to Markdown)?

Reply:
It's so that it can then be referenced from other documents (via `:ref:nvidia-gpu`, for example in the `Next Steps` section of another doc)